CRM/Utils/Check/Component/Env.php

```diff
@@ -606,7 +605,7 @@ public function checkExtensions() {
         __FUNCTION__,
         ts('There are currently no extensions on the CiviCRM public extension directory which are compatible with version %1. If you want to install an extension which is not marked as compatible, you may be able to <a %2>download and install extensions manually</a> (depending on access to your web server).', [
```

Comment:
I think this will need to be changed so its 
```suggestion
        ts('There are currently no extensions on the CiviCRM public extension directory which are compatible with version %1. If you want to install an extension which is not marked as compatible, you may be able to <a href="%2">download and install extensions manually</a> (depending on access to your web server).', [
```

Reply:
I think the original is correct, or at least the civi standard, because it allows adding/removing/changing the attributes without breaking already translated strings. See https://docs.civicrm.org/dev/en/latest/translation/#avoid-tags-inside-strings where it talks about hyperlinks inside big text blocks.

Reply:
I guess my thinking is that wit the proposed change the href= is getting droped in param 2 so it needs to either get restored there or be added here

Reply:
Oh ok I see. Then it would be `<a %2>` and the href would be added into array param 2.

Reply:
You're right, it's lost the href.  Changed now to default form